docker-compose.yml

```diff
@@ -1,38 +1,64 @@
 version: '3.8'
 
 services:
-  nginx:
-    container_name: badge-poser-nginx
-    image: nginx:alpine
+  app:
+    container_name: badge-poser-app
+    build:
+      dockerfile: sys/docker/Dockerfile
+      context: .
+      network: host
     working_dir: /application
-    depends_on:
-      - phpfpm
     volumes:
       - .:/application:cached
     ports:
       - 8001:80
-
-  phpfpm:
-    build:
-      context: docker/dev/php-fpm
-      network: host
-    container_name: badge-poser-phpfpm
-    working_dir: /application
     depends_on:
       - redis
-    volumes:
-      - .:/application:cached
-      - ./docker/dev/php-fpm/php-ini-overrides.ini:/etc/php/7.4/fpm/conf.d/99-overrides.ini:ro
-    ports:
-      - 9000
+    deploy:
+      resources:
+        limits:
+          cpus: '0.5'
+          memory: 1024M
 
-  node:
-    container_name: badge-poser-node
-    image: node:12-alpine
-    user: "node"
-    volumes:
-      - .:/application:cached
-    working_dir: /application
+#  nginx:
```

Comment:
Is it important to keep these comment lines? I would suggest removing those. What do you think?